packages/jitm/src/class-pre-connection-jitm.php

```diff
@@ -33,6 +36,7 @@ private function get_raw_messages() {
 				'description'    => __( 'Set up Jetpack to get in-depth stats about your content and visitors.', 'jetpack' ),
 				'button_link'    => $jetpack_setup_url,
 				'button_caption' => __( 'Set up Jetpack', 'jetpack' ),
+				'plugin_active'  => array( 'jetpack/jetpack.php' ),
```

Comment:
Should we take the Jetpack Beta plugin into consideration here, and add `jetpack-dev`?

Reply:
Good catch! Yes, we should definitely allow the pre-connection JITMs to display with the Beta plugin. As I mentioned in the comment above, maybe I should take a totally different approach to this, where a list of plugin slugs won't be necessary.